### PR TITLE
Updating comment options order - Fixes Issue #2833

### PR DESCRIPTION
**Pull request by Neidley**
screen shot 2018-06-16 at 6 55 40 pm

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -114,7 +114,7 @@ GEM
       activerecord (>= 4.0.0)
     gemoji (3.0.0)
```
Comment (Gauravano) [inline Gemfile.lock]: this file need to be excluded!

File: app/views/notes/_comment.html.erb [hunk 2]
```diff
@@ -23,26 +23,6 @@
...
     </div>
 
-      <div class="hidden-print btn-toolbar pull-right" style="margin-left:10px;">
-        <ul class="btn-group" style="margin-top: 16px; margin-right: 16px;">
-          <% if !current_user %>
-            <li rel="tooltip" title="Helpful? Like it and get updates!" class="btn btn-default btn-sm btn-like">
-            <%= link_to  new_user_session_path( return_to: request.path ), style: "text-decoration: none; color: black;" do %>
-              <span class="fa fa-star-o"></span>
-              <span><%= comment.likers.count %></span>
-            <% end %>
```
Comment (Gauravano) [inline app/views/notes/_comment.html.erb]: Awesome work here!

File: www.example.com/db/openid-store/associations/http-localhost-Sve98kiqQxHfzN8CdZK0Uuu1aIs-90pW6t.dYrSWApdWm3Ns5V_25pc [hunk 3]
```diff
@@ -0,0 +1,6 @@
+version:2
+handle:{HMAC-SHA1}{5b259d93}{TgWPDA==}
+secret:QmiQg8McY3Z95zIZvXEBtPl23A0=
```
Comment (Gauravano) [inline www.example.com/db/openid-store/associations/http-localhost-Sve98kiqQxHfzN8CdZK0Uuu1aIs-90pW6t.dYrSWApdWm3Ns5V_25pc]: this file is also not needed

File: www.example.com/db/openid-store/associations/http-localhost-Sve98kiqQxHfzN8CdZK0Uuu1aIs-JbRDRpXM2E2.VCy3cFsI5CJWF2k [hunk 4]
```diff
@@ -0,0 +1,6 @@
+version:2
```
Comment (Gauravano) [inline www.example.com/db/openid-store/associations/http-localhost-Sve98kiqQxHfzN8CdZK0Uuu1aIs-JbRDRpXM2E2.VCy3cFsI5CJWF2k]: Same for this file

Comment (Neidley) [reply www.example.com/db/openid-store/associations/http-localhost-Sve98kiqQxHfzN8CdZK0Uuu1aIs-JbRDRpXM2E2.VCy3cFsI5CJWF2k]: Thanks for all your help and for my first PR 👍

**Comment by jywarren** [comment]
Oh, oops! There's no need to close this PR if you're still working on it - you can keep adding commits and we'll be here to help out. I'll re-open it unless there's a new place you're working? 

Thanks for your help!

**Comment by jywarren** [comment]
Oh, ok! Oops! I see this is now over at https://github.com/publiclab/plots2/pull/2834#issuecomment-397816391

Closing again! Thanks!

**Pull request closed without merging**